### PR TITLE
Bugreport918 vertical spacing in side menu

### LINKED ISSUE #918: [Bug Report] Main menu with a larger vertical separation between items
#### Steps to reproduce

1. Login

#### Screenshot or Gif（截图或动态图）
sidebar-menu-error

#### Expected behavior
The main menu is displayed with a vertical spacing between items larger than the correct spacing.

#### Other relevant information
* Your OS: Linux Mint 19.1 Cinnamon x64. 
* Browser: Mozilla Firefox 88.0.1.
* Node.js version: 14.17.0.
* NPM version: 7.17.0.
* adempiere-vue version: 4.3.1.

#### Additional context
This problem apparently only affects Mozilla Firefox, in Google Chrome it is displayed correctly.

### PR DESCRIPTION
**Pull request by sofia-ester-calderon**
Corrected the big vertical spacing in the side menu (issue only present in Firefox).

## Bug report / Feature

Closes #918 

#### Screenshot or Gif

**Firefox:**
Screen Shot 2021-06-16 at 12 07 29 pm

**Chrome:**
Screen Shot 2021-06-16 at 12 08 04 pm

### CODE REVIEW

**Comment by sofia-ester-calderon** [comment]
also fixed the styling of sidbar when menu is collapsed

**Before:**
Screen Shot 2021-06-17 at 3 15 00 pm

**After:**

- Firefox:
Screen Shot 2021-06-17 at 3 10 28 pm

- Chrome:
Screen Shot 2021-06-17 at 3 10 00 pm

- Mobile:
Screen Shot 2021-06-17 at 3 11 28 pm

**Comment by sofia-ester-calderon** [comment]
Also fixed the style of collapsed sidebar menu.

**Before**
Screen Shot 2021-06-17 at 3 15 00 pm


**After**

- Firefox
Screen Shot 2021-06-17 at 6 08 03 pm

- Chrome
Screen Shot 2021-06-17 at 6 08 25 pm

- Mobile
Screen Shot 2021-06-17 at 6 11 09 pm

File: src/layout/components/Sidebar/Item.vue [hunk 1]
```diff
@@ -22,25 +21,56 @@ export default {
...
-  white-space: break-spaces;
-  line-height: 20px;
-  height: auto;
-  margin: 18px 23px 18px 0;
-}
+  .item-wrapper {
+    white-space: break-spaces;
+    line-height: 20px;
+    height: auto;
+    padding: 18px 23px 18px 0;
```
Comment (EdwinBetanc0urt) [inline src/layout/components/Sidebar/Item.vue]: Please delete the spaces at the end of the line 58 as it gives a warning message with the eslint

compile-with-error

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

**Comment by EdwinBetanc0urt** [comment]
Hi @sofia-ester-calderon, I ran the tests and they were successful on both firefox desktop for linux and firefox mobile for android.

Mozilla Firefox desktop:
https://user-images.githubusercontent.com/20288327/122490786-c0c10100-cfb0-11eb-86e3-ebc39b68726f.mp4


Mozilla Firefox mobile:

https://user-images.githubusercontent.com/20288327/122490841-dfbf9300-cfb0-11eb-8abd-c0ba2647392e.mp4


@yamelsenih  please merge this pull request.

**Review by EdwinBetanc0urt — APPROVED** [review]

**Comment by yamelsenih** [comment]
Hi @sofia-ester-calderon already exist some changes by @elsiosanchez for SidebarItem, can you review and resolve conflicts?

Best regards

**Comment by sofia-ester-calderon** [comment]
> Hi @sofia-ester-calderon already exist some changes by @elsiosanchez for SidebarItem, can you review and resolve conflicts?
> 
> Best regards

Done! Happy weekend :)

**Comment by yamelsenih** [comment]
Great!!! Thanks... Me too

**Pull request merged**